pkgs/applications/misc/auto-multiple-choice/default.nix

```diff
@@ -75,10 +75,12 @@ stdenv.mkDerivation rec {
       GlibObjectIntrospection
       Gtk3
       LocaleGettext
+      OpenOfficeOODoc
       PerlMagick
       TextCSV
       XMLParser
       XMLSimple
+      XMLTwig
```

Comment:
Is this a dependency of AMC or OpenOfficeOODoc?

Reply:
Yes, this is a dependency of OODoc. I wrote in the commit message that it doesn't work without it. Based on your comment, I looked at how pulling of transitive dependencies works for Perl and it seem that we should use `makeFullPerlPath` instead of `makePerlPath`. I've pushed an updated commit with this change.